src/connectedk8s/azext_connectedk8s/vendored_sdks/_version.py

```diff
@@ -6,4 +6,4 @@
 # Changes may cause incorrect behavior and will be lost if the code is regenerated.
 # --------------------------------------------------------------------------
 
-VERSION = "1.0.0"
+VERSION = "1.0.0b1"
```

Comment:
Why is this version looking like this ?. Compared to the before one ?.

Reply:
we haven't made this change. track-2 sdk team made this change.